Library/Message.swift

```diff
@@ -1,74 +1,190 @@
-                }
-            }
-
-            self.init(rawValue: kindRawValue)!
-        }
-    }
-
-    init(format: String) {
-        self.format = format
     }
-
-    let format: String
-
+    
+    private let summerSolsticeMessages = [
+        "Happy summer solstice! Get yourself a healthy summer glow.",
+        "Happy first day of summer: make the most out of it!"
+    ]
+    
+    private let winterSolsticeMessages = [
+        "Happy winter solstice! The light begins its journey back to full brightness from now on.",
+        "Have a magical winter solstice! The light will soon brighten up your days again."
+    ]
+    
+    private let messages: [SunTime.DayOrNight: [TimeDiferential: [String]]] = [
+        .day: [
+            .longerMoreThanAMinute: [
+                "Today is %@ longer than yesterday. Happy days!",
+                "The sun is out for %@ more today. Enjoy!",
+                "%@ extra sunshine today. Make them count!",
+                "Make sure to soak up that vitamin D. %@ more daylight today!",
+                "Smile! Today has %@ more daylight than yesterday!",
+                "%@ more daylight today. Just let it sink in…",
+                "Today is %@ longer. It’s getting better and better!",
+                "Bring out your shorts, because today has %@ more sunlight.",
+                "Have a great day and enjoy those %@ extra daylight.",
+                "After darkness comes daylight. %@ more to be precise!"
+            ],
+            .longerLessThanAMinute: [
+                "Little less than a minute of extra sunlight today. It’s getting better!",
+                "We’ve reached the tipping point: we’ll have more sunlight every day now!",
+                "About a minute of extra light. You’ll start noticing the difference soon!",
+                "There’s about a minute of extra light at the end of this tunnel.",
+                "We’ll have about a minute of extra light today. It’s upwards from here."
+            ],
+            .shorterMoreThanAMinute: [
+                "The sun will be out ** %@ less** today. Keep your head up!",
+                "%@ less sunlight today, unfortunately. It’ll get better!",
+                "Sadly, the day will be %@ shorter. Make the most out of it!"
+            ],
+            .shorterLessThanAMinute: [
+                "Unfortunately, the day is a little bit shorter today. Make the most out of it!",
+                "Sadly, today is a tiny bit shorter than yesterday. Enjoy it while it lasts!",
+                "Today is shorter than yesterday. But fear not, brighter times ahead!"
+            ]
+        ],
+        .night: [
+            .longerMoreThanAMinute: [
+                "Get a good night’s sleep: tomorrow there’ll be %@ more sunlight.",
+                "Lights out. Enjoy %@ more sunlight tomorrow!",
+                "Bring out those pyjamas. ** %@** more light await tomorrow.",
```

Comment:
Whoops missed this earlier: `awaits`

Reply:
...i think? `two minutes more light await tomorrow` vs `two minutes more light awaits tomorrow`  - `awaits` sounds better to me but  even as a native english speaker I'm not 100% sure on this one.